.env.example

```diff
@@ -1,6 +1,7 @@
 TIME_PERIOD_DAYS=7
 
 GITHUB_ORG_NAME=
+GITHUB_REPO_NAME=
```

Comment:
Nevo mentioned "`You will check all the repositories for the org (with GitHub GraphQL) for the most amount of PR for a single contributor.`", single repo might not work :D

Reply:
It is still under work, bruh :D

Reply:
@archaeopteryx1 I have noted the change with a warning in my comment. Should resolve any upcoming conversations.